BUILD.bazel

```diff
@@ -49,6 +50,20 @@ genrule(
     cmd = "mkdir $@",
 )
 
+genrule(
+    name = "copy_extend_oem_bin",
+    srcs = ["//tools/cmd/extend_oem:main_for_build"],
+    outs = ["extend-oem.bin"],
+    cmd = "mv $< $@",
+)
+
+pkg_tar(
+    name = "remap_extend_oem_bin",
+    srcs = ["copy_extend_oem_bin"],
```

Comment:
I'm not sure why this works; but "copy_extend_oem_bin" should instead be ":copy_extend_oem_bin", since you are referencing the output of another target.